custom-recipes/timeseries-preparation-resampling/recipe.json

```diff
@@ -264,9 +264,49 @@
       ],
       "defaultValue": "clip"
     },
+    {
+      "name": "category_imputation_method",
+      "label": "Impute categoricals",
```

Comment:
I guess it should be either categoricals or categorical data because categorical is just an adjective.  I heard Krishna say "categoricals" yesterday but maybe it's too informal

Reply:
done in https://github.com/dataiku/dss-plugin-timeseries-preparation/pull/21/commits/3a4f3a6aafa82c2fb8b8cf7b2514f21f61ed3edc

Reply:
changed in https://github.com/dataiku/dss-plugin-timeseries-preparation/pull/21/commits/3a4f3a6aafa82c2fb8b8cf7b2514f21f61ed3edc